RELEASE.md

```diff
@@ -50,18 +50,50 @@ This release contains contributions from:
 
 ## Major Features and Improvements
 
+* <INSERT MAJOR FEATURE HERE, USING MARKDOWN SYNTAX>
+* <IF RELEASE CONTAINS MULTIPLE FEATURES FROM SAME AREA, GROUP THEM TOGETHER>
+
+## Bug Fixes and Other Changes
+
+* <SIMILAR TO ABOVE SECTION, BUT FOR OTHER IMPORTANT CHANGES / BUG FIXES>
+* <IF A CHANGE CLOSES A GITHUB ISSUE, IT SHOULD BE DOCUMENTED HERE>
+* <NOTES SHOULD BE GROUPED PER AREA>
+
+## Thanks to our Contributors
+
+This release contains contributions from:
+
+<INSERT>, <NAME>, <HERE>, <USING>, <GITHUB>, <HANDLE>
+
+
+# Release 2.7.0
+
+The main theme of this release is documentation, with a new suite of tutorials, several upgrades to notebooks and the removal of a rather annoying bug in the documentation site.
+
+Perhaps more notably, `check_shapes` has been removed, and can now be found [here](https://github.com/GPflow/check_shapes).  This change is breaking for those who are still getting `check_shapes` from `gpflow`.  This intended to be the last release with breaking changes that is not a major version number; starting with the next release we will be moving to semantic versioning.
```

Comment:
(I mean check_shapes _was_ in experimental)

Reply:
That's a good point.  Let me reword this.